app/backend/src/clj/teet/meeting/meeting_commands.clj

```diff
@@ -204,15 +205,52 @@
          "/meetings/" activity-eid
          "/" meeting-eid)))
 
+
+(defn historical-meeting-notify [db meeting from-user participants project-eid]
+  (log/debug "from-user is" from-user)
+  (tx-ret
+   (vec
+    (for [to-user participants]
+      (notification-db/notification-tx
+       db
+       (do
+         (log/debug "to-user is" to-user)
+           
+         {:from from-user
+          :to to-user
+          :target (:db/id meeting)
+          :type :notification.type/meeting-updated
+          :project project-eid}))))))
+
+(defn send-meeting-email! [db meeting project-eid meeting-link to meeting-eid]
+  (let [meeting-link (meeting-link db
+                          (environment/config-value :base-url)
+                          meeting project-eid)
+            ics (meeting-ics/meeting-ics {:meeting meeting
+                                          :meeting-link meeting-link
+                                          :cancel? false})
+            email-response
+            (integration-email/send-email!
+              {:from (environment/ssm-param :email :from)
+               :to to
+               :subject (str "TEET: koosoleku kutse " (meeting-model/meeting-title meeting) " / "
```

Comment:
this has to change for the past-meeting case because it's no longer an invitation